securedrop_client/crypto.py

```diff
@@ -48,6 +48,7 @@ def decrypt_submission_or_reply(filepath, target_filename, home_dir,
             logger.error("GPG error: {}".format(msg))
 
         os.unlink(err.name)
+        os.unlink(out.name)
```

Comment:
how about we can move these two lines to right before we return from this function? so that in both cases these files are cleaned up